### PR TITLE
Adding a file - jarvey33 - Create a file for githubteacher repo

### LINKED ISSUE #7: jarvey33's recommendations in Chicago
## Goose Island

Take the brewery tour and test out some good brews. :beers:

### PR DESCRIPTION
**Pull request by jarvey33**
Closes #7

### CODE REVIEW

File: jarvey33-chicago.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Fun stuff to do in Riverwoods
+
+Nothing
+
+## Great Places to Eat
+
+Big Bowl
+Real Urban Barbecue
```
Comment (brianamarie) [inline jarvey33-chicago.md:8]: Barbecue sounds delicious! What's your favorite thing to order there?

File: jarvey33-chicago.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+## Fun stuff to do in Riverwoods
+
+Nothing
+
+## Great Places to Eat
```
Comment (snalla100) [inline jarvey33-chicago.md:5]: :100:

File: jarvey33-chicago.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+## Fun stuff to do in Riverwoods
+
+Nothing
```
Comment (richavekaria) [inline jarvey33-chicago.md:3]: Have fun!!

Comment (richavekaria) [inline jarvey33-chicago.md:3]: this is interesting

File: jarvey33-chicago.md (hunk 1 above)
Comment (jarvey33) [inline jarvey33-chicago.md:8]: I love the beef brisket sandwich!

File: jarvey33-chicago.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+## Great Places to Eat
+
+Big Bowl
+Real Urban Barbecue
+Jason's Deli
+
+## Fun things to do
+
+Leave Riverwoods
```
Comment (jlestes) [inline jarvey33-chicago.md:13]: Hahaha! :laughing:

**Pull request merged**